src/core/Microsoft.PowerToys.Settings.UI/Views/KeyboardManagerPage.xaml

```diff
@@ -0,0 +1,244 @@
+                    </VisualState.Setters>
+                </VisualState>
+                <VisualState x:Name="SmallLayout">
+                    <VisualState.StateTriggers>
+                        <AdaptiveTrigger MinWindowWidth="{StaticResource SmallLayoutMinWidth}" />
+                    </VisualState.StateTriggers>
+                    <VisualState.Setters>
+                        <Setter Target="SidePanel.(Grid.Column)" Value="0" />
+                        <Setter Target="SidePanel.(Grid.Row)" Value="1" />
+                    </VisualState.Setters>
+                </VisualState>
+            </VisualStateGroup>
+        </VisualStateManager.VisualStateGroups>
+        <Grid.ColumnDefinitions>
+            <ColumnDefinition Width="*"/>
+            <ColumnDefinition Width="Auto"/>
+        </Grid.ColumnDefinitions>
+        <Grid.RowDefinitions>
+            <RowDefinition Height="Auto"/>
+            <RowDefinition Height="Auto" />
+        </Grid.RowDefinitions>
+
+        <StackPanel Orientation="Vertical">
+            <TextBlock Text="Reconfigure your keyboard by remapping keys and shortcuts"
+                       TextWrapping="Wrap"
+                       Style="{StaticResource SettingsGroupTitleStyle}"/>
+
+            <Grid Margin="{StaticResource SmallTopMargin}">
+                <TextBlock Text="Current configuration"
+                           Style="{StaticResource SubtitleTextBlockStyle}" />
+                <ComboBox SelectedIndex="1"
+                          Margin="{StaticResource ResourceKey=MediumRightMargin}"
+                          HorizontalAlignment="Right">
+                    <ComboBoxItem Content="Config-1"/>
+                    <ComboBoxItem Content="Config-2"/>
+                    <ComboBoxItem Content="Config-3"/>
+                </ComboBox>
+            </Grid>
+
+            <TextBlock Text="Select the profile to display the active key remap and shortcuts"
+                Margin="{StaticResource XSmallTopMargin}"/>
+
+            <Grid Margin="{StaticResource SmallTopMargin}">
+                <TextBlock Text="Remap Keyboard"
+                           Style="{StaticResource SubtitleTextBlockStyle}" />
+                <Button Content="Remap"
+                        Margin="{StaticResource MediumLeftRightMargin}"
+                        Style="{StaticResource AccentButtonStyle}"
+                        HorizontalAlignment="Right"/>
+            </Grid>
+
+            <TextBlock Text="To remap keys, press the remap button"
+                       Margin="{StaticResource XSmallTopMargin}"/>
+
+            <StackPanel Margin="{StaticResource SmallLeftTopRightBottomMargin}"
+                        Orientation="Horizontal">
+                <TextBlock Text="Remap for:"
+                           FontWeight="Bold"/>
+                <TextBlock Text="Config-1"
+                           Margin="{StaticResource XSmallLeftMargin}"
+                           FontWeight="ExtraLight" />
```

Comment:
```suggestion
                           Foreground="{StaticResource SystemAccentColorBrush" />
```
Maybe, instead of using ExtraLight (which is meant for large titles due to readability issues on smaller fontsizes) we could use the accent color here so it stands out more?

Reply:
This is no longer applicable because I removed the UI element itself as per the suggestions below.